### DISCUSSION TITLE
Where can I find TSM tutorial?

### DISCUSSION THREAD

**Discussion by unbob** [sienori/Tab-Session-Manager#1245] [Q&A]
Surely somewhere there is a tutorial fully explaining how to use TSM. Where?

I'm a TSM newbie and have tried to use it but have found it to be "scary" - I've lost tabs and windows and been unable to recover.

TSM appears to be an impressive and very useful add-on but complete documentation is absolutely needed.

TIA

**Comment by matrixmainframe** [comment]
Have to agree.  I've installed the Tab Session Manager but have no idea how it works or how to use it or even to open it??!!!!!

**Comment by muel85** [comment]
+1 on that . . .
. . . this is how I came to be here, looking for help documentation.


I'm just trying to figure out what the following means exactly in the settings  . . .

'Save the session when window was closed
(Save the session that includes only one closed window.)'

. . . and what the following means exactly and how it differs from the above . . .

'Save the session when exiting browser
(Save the session that includes multiple windows that was closed in a row.)'


When 'window' is mentioned, is it referring to 'tabs'? As for me you 'exit' the browser by closing the 'browser window'.
. . . and what does 'closed in a row' mean?

**Comment by muel85** [comment]
I figured some of it out, seems so obvious now.

I forgot to consider you can have multiple window instances of a browser, as well as multiple tabs.
